frontend/packages/console-shared/src/components/catalog/CatalogController.tsx

```diff
@@ -51,13 +52,28 @@ const CatalogController: React.FC<CatalogControllerProps> = ({
   const { pathname } = useLocation();
   const queryParams = useQueryParams();
 
-  const typeExtension = React.useMemo(
+  const typeExtension: ResolvedExtension<CatalogItemType> = React.useMemo(
     () => catalogExtensions?.find((extension) => extension.properties.type === type),
     [catalogExtensions, type],
   );
-
   const title = typeExtension?.properties.title ?? defaultTitle;
-  const description = typeExtension?.properties.catalogDescription ?? defaultDescription;
+
+  const getCatalogDescription = (): JSX.Element => {
+    if (typeof typeExtension?.properties?.catalogDescription === 'string') {
+      return (
+        <p className="co-catalog-page__description">
+          {typeExtension?.properties?.catalogDescription}
+        </p>
+      );
+    }
+    if (
+      typeof typeExtension?.properties?.catalogDescription !== 'string' &&
```

Comment:
Mini nit: If it's a string above, we can skip this check. If typescript needs this I'm also fine with this. This is btw the reason I'm a big fan of else if statements. But our prettier prefers if statements :smile: :shrug: 

```suggestion
```

Reply:
Fixed it